src/components/editor/ComponentPreview.tsx

```diff
@@ -46,8 +48,9 @@ const importView = (component: any) => {
 }
 
 const ComponentPreview: React.FC<{
-  componentName: string
-}> = ({ componentName, ...forwardedProps }) => {
+  componentName: string,
+  item?: string
```

Comment:
can we extend the solution using forwarded props? as we can have nested items as well

Reply:
I'll experiment with this also